src/main/kotlin/org/jitsi/jicofo/conference/colibri/v2/ColibriV2SessionManager.kt

```diff
@@ -360,63 +359,70 @@ class ColibriV2SessionManager(
                     // Most probably we sent a bad request.
                     // If we flag the bridge as non-operational we may disrupt other conferences.
                     // If we trigger a re-invite we may cause the same error repeating.
-                    throw BadColibriRequestException(response.error?.toXML()?.toString() ?: "null")
+                    throw ColibriAllocationFailedException("Bad request: ${response.error?.toXML()?.toString()}", false)
                 }
                 item_not_found -> {
                     if (reason == Colibri2Error.Reason.CONFERENCE_NOT_FOUND) {
                         // The conference on the bridge has expired. The state between jicofo and the bridge is out of
                         // sync.
-                        throw ColibriConferenceExpiredException(session.bridge, true)
+                        throw ColibriAllocationFailedException("Conference not found", true)
                     } else {
                         // This is an item_not_found NOT coming from the bridge. Most likely coming from the MUC
                         // because the occupant left.
-                        throw BridgeFailedException(session.bridge, true)
+                        throw ColibriAllocationFailedException("Item not found, bridge unavailable?", true)
                     }
                 }
                 conflict -> {
                     if (reason == null) {
                         // An error NOT coming from the bridge.
-                        throw BridgeFailedException(session.bridge, true)
+                        throw ColibriAllocationFailedException(
+                            "XMPP error: ${response.error?.toXML()}",
+                            true
+                        )
                     } else {
                         // An error coming from the bridge. The state between jicofo and the bridge must be out of sync.
                         // It's not clear how to handle this. Ideally we should expire the conference and retry, but
                         // we can't expire a conference without listing its individual endpoints and we think there
                         // were none.
```

Comment:
Is this something we should fix in colibri2?

Reply:
Yeah, I think so.

Reply:
I can do that in a seperate set of PRs